docs/products/opensearch/concepts/opensearch_backups.rst

```diff
@@ -17,7 +17,10 @@ Depending on the service plan, we offer single backups for disaster recovery or
 To allow returning to an earlier point in time, Aiven for OpenSearch uses two kinds of backups - hourly and daily. Each type has its own
 backup frequency interval and retention period. Currently, you cannot configure these settings.
 
-Aiven for OpenSearch uses the database forking functionality to create a new OpenSearch service that is cloned from the backup that you select.
+Restore from backup
```

Comment:
This kind of duplicates the content in opensearch/howto/restore_opensearch_backup - I'd suggest removing one or the other

Reply:
Will do